lib/timers.js

```diff
@@ -483,152 +577,3 @@ exports.clearImmediate = function(immediate) {
     process._needImmediateCallback = false;
   }
 };
-
-
-// Internal APIs that need timeouts should use timers._unrefActive instead of
-// timers.active as internal timeouts shouldn't hold the loop open
-
-var unrefList, unrefTimer;
-
-function _makeTimerTimeout(timer) {
-  var domain = timer.domain;
-  var msecs = timer._idleTimeout;
-
-  L.remove(timer);
-
-  // Timer has been unenrolled by another timer that fired at the same time,
```

Comment:
> because we used to queue all internal/unref timers that need to timeout.

doh, that would make it necessary yes

> So it seems that the comment at https://github.com/nodejs/node/pull/4007/files#diff-0a5d4868b2b9b17cf9e2c11f1bd1311eR130 about the `listOnTimout` loop timer being "too early" because of unenrolled timers is not accurate, and should be removed, because these unenrolled timers won't even be processed in this loop.

Yeah I caught that earlier but forgot to remove the comment. D: